content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,109 @@
+
+We also have added an [authors page](/authors) that's very similar. On each author's page, you'll find their title for rOpenSci staff members, their online home, possibly their Twitter handle, and all the content they've authored on our website. [See for instance Stefanie Butland's author page](authors/stefanie-butland). These pages are linked from the bottom of posts for returning authors. Thanks to [Angela Li](https://angela-li.github.io/) for [requesting we added authors pages](https://github.com/ropensci/roweb2/issues/365).
+
+Technically, getting tags and authors page and tag pages is quite straightforward with Hugo thanks to _taxonomies_. To switch them on, we added these lines to the website config.toml
+
+```
+[taxonomies]
+  tag = "tags"
+  author = "authors"
+```
+
+We added this option as well
+
+```
+preserveTaxonomyNames = true
+```
+
+to not mess up with authors' names capitalization when listing them.
+
+For authors, a challenge was moving metadata around, from storing authors' metadata (online home, Twitter username) inside posts metadata to having authors metadata files under content/authors, see [this example](https://github.com/ropensci/roweb2/blob/master/content/authors/ignasi-bartomeus/_index.md), but as mentioned earlier, with the `yaml` package you can edit hundreds of Markdown files at once, granted you check the edits carefully.
+
+Read more about [Hugo taxonomies](https://gohugo.io/content-management/taxonomies/), how to write [templates for taxonomy pages](https://gohugo.io/templates/taxonomy-templates/) and [where to save them so Hugo finds them](https://gohugo.io/templates/lookup-order/#examples-layout-lookup-for-taxonomy-list-pages). In our case, the tags and authors pages have different templates: [tags page](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/terms.html), [tag pages](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/taxonomy/tag.html), [authors templates](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts/authors)
+
+### Archive
+
+With precious content scattered over three categories (blog, tech notes, community calls), getting an overview can be tricky hence our adding an [archive page](/archive) featuring all our website content [from our very first blog post](https://ropensci.org/blog/2012/11/26/is-invasive/). Behind the scenes, we [list pages with a date](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html#L29) and [feeds them to an archive partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/archive.html) that we use for the [technotes page](/technotes) and authors pages as well for consistent presentation.
```

Comment:
I think for less experienced readers, saying "we list pages with a date" may be confusing